<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WallpaperSettingsDialog</class>
 <widget class="QDialog" name="WallpaperSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wallpaper Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>730</y>
     <width>821</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>221</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse Image</string>
   </property>
  </widget>
  <widget class="QComboBox" name="desktop">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="desktopLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Desktop</string>
   </property>
  </widget>
  <widget class="QLabel" name="screenLabel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Screen</string>
   </property>
  </widget>
  <widget class="QComboBox" name="screen">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>91</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="previewHolder">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>821</width>
     <height>521</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="preview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="previewLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WallpaperSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WallpaperSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
